learning_journal/learning_journal/views.py

```diff
@@ -0,0 +1,32 @@
+from pyramid.view import view_config
+
+
+@view_config(
+    route_name='home', renderer='learning_journal:templates/home.html'
+)
```

Comment:
This is a step further than we wanted you to go.  But that's okay.  No worries.  You're set up nicely for your work in step 2